agg.pretty/ultra-fit-01x03.kicad_mod

```diff
@@ -0,0 +1,24 @@
+(module ultra-fit-01x03 (layer F.Cu) (tedit 5A85C304)
+  (fp_text reference REF** (at 0 -8) (layer F.Fab)
+    (effects (font (size 1 1) (thickness 0.15)))
+  )
+  (fp_text value ultra-fit-01x03 (at 0 4.5) (layer F.Fab)
+    (effects (font (size 1 1) (thickness 0.15)))
+  )
+  (fp_line (start -6.5 -7) (end -6.5 3.5) (layer F.CrtYd) (width 0.01))
+  (fp_line (start 6.5 -7) (end -6.5 -7) (layer F.CrtYd) (width 0.01))
+  (fp_line (start 6.5 3.5) (end 6.5 -7) (layer F.CrtYd) (width 0.01))
+  (fp_line (start -6.5 3.5) (end 6.5 3.5) (layer F.CrtYd) (width 0.01))
+  (fp_line (start -5.75 2.93) (end 5.75 2.93) (layer F.SilkS) (width 0.15))
+  (fp_line (start 5.75 2.93) (end 5.75 -2.42) (layer F.SilkS) (width 0.15))
+  (fp_line (start 5.75 -2.42) (end 3.645 -2.42) (layer F.SilkS) (width 0.15))
+  (fp_line (start 3.645 -2.42) (end 3.645 -5.48) (layer F.SilkS) (width 0.15))
+  (fp_line (start 3.645 -5.48) (end -3.645 -5.48) (layer F.SilkS) (width 0.15))
+  (fp_line (start -3.645 -5.48) (end -3.645 -2.42) (layer F.SilkS) (width 0.15))
+  (fp_line (start -3.645 -2.42) (end -5.75 -2.42) (layer F.SilkS) (width 0.15))
+  (fp_line (start -5.75 -2.42) (end -5.75 2.93) (layer F.SilkS) (width 0.15))
+  (pad "" np_thru_hole circle (at -1.75 2.23) (size 1.6 1.6) (drill 1.6) (layers *.Cu *.Mask))
+  (pad 3 thru_hole circle (at 3.5 0) (size 2 2) (drill 1.06) (layers *.Cu *.Mask))
+  (pad 2 thru_hole circle (at 0 0) (size 2 2) (drill 1.06) (layers *.Cu *.Mask))
+  (pad 1 thru_hole circle (at -3.5 0) (size 2 2) (drill 1.06) (layers *.Cu *.Mask))
+)
```

Comment:
Please add a drawing of the part on F.Fab - this can be the same as the silk but 0.01mm thick.